src/onetime/entrypoints/web/web_flask/containers.py

```diff
@@ -0,0 +1,13 @@
+from dependency_injector import containers, providers
+from onetime.services.manager import SecretManager
+from onetime.use_cases.manager import SecretAndUrlManager
+
+
+class Container(containers.DeclarativeContainer):
```

Comment:
No need for a separate container.
Please use what we have in `configurator/contianers`